desktop/src/player.rs

```diff
@@ -202,7 +202,19 @@ impl ActivePlayer {
             RfdNavigatorInterface,
         );
 
-        if cfg!(feature = "software_video") {
+        if cfg!(feature = "external_video") && preferences.openh264_enabled() {
+            use ruffle_video_external::backend::ExternalVideoBackend;
```

Comment:
I appreciate the effort to feature gate this, but this is a compile error if the feature is turned off.

Reply:
Good catch, thank you! Added an inner block that is conditionally compiled.